packages/api-reference/src/components/Content/Content.vue

```diff
@@ -36,50 +102,88 @@ const clientOptions = computed(() =>
     <slot name="start" />
 
     <!-- Introduction -->
-    <Introduction
-      v-if="document?.info?.title || document?.info?.description"
-      :document
-      :config>
-      <template #serverSelector>
-        <div
-          v-if="store.workspace.activeDocument?.servers?.length"
-          class="scalar-reference-intro-server scalar-client introduction-card-item text-base leading-normal [--scalar-address-bar-height:0px]">
-          <ServerSelector
-            :servers="store.workspace.activeDocument?.servers ?? []"
-            :xSelectedServer="
-              store.workspace.activeDocument?.['x-scalar-active-server']
-            " />
-        </div>
-      </template>
-      <template #clientSelector>
-        <ClientSelector
-          v-if="config?.hiddenClients !== true && clientOptions.length"
-          :clientOptions
-          :xScalarSdkInstallation="
-            store.workspace.activeDocument?.info?.['x-scalar-sdk-installation']
-          "
-          :xSelectedClient="store.workspace['x-scalar-default-client']"
-          class="introduction-card-item scalar-reference-intro-clients" />
-      </template>
-    </Introduction>
+    <IntroductionSection :showEmptyState="!store.workspace.activeDocument">
+      <InfoBlock
+        v-if="store.workspace.activeDocument"
+        :id
+        :documentExtensions
+        :externalDocs="store.workspace.activeDocument.externalDocs"
+        :getOriginalDocument
+        :info="store.workspace.activeDocument.info"
+        :infoExtensions
+        :isLoading="config.isLoading"
+        :layout="config.layout"
+        :oasVersion="store.workspace.activeDocument['x-original-version']"
```

Comment:
can we pls use

`x-original-oas-version`

because there is a `version` property, too